src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --light-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: block;
+}
+
+.nav__list {
+  padding-inline-start: 0;
+  display: inline;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+/* a {
+  
+} */
+
+.links {
+  width: 636px;
+  text-align: right;
+  height: 60px;
+  line-height: 60px;
+  list-style-type: none;
```

Comment:
Avoid using 'list-style-type' on a class that is not directly related to list items. Instead, apply list-related styles directly to the '.nav__list' class or its 'li' children.